src/solvers/flattening/boolbv.cpp

```diff
@@ -523,64 +498,52 @@ literalt boolbvt::convert_rest(const exprt &expr)
     return convert_overflow(expr);
   else if(expr.id()==ID_isnan)
   {
-    if(expr.operands().size()!=1)
-      throw "isnan expects one operand";
-
+    INVARIANT(expr.operands().size() == 1, "isnan expects one operand");
     const bvt &bv=convert_bv(operands[0]);
-
```

Comment:
Could you please refrain from removing those blank lines? It really hurts readability. Thanks.

Reply:
I think it improves readability, as less code sits outside of the screen/peripheral vision. The more spaced out the text is, the less legible it gets, relations between variables are harder to notice, etc.

Reply:
Whatever. Changing whitespace has no place in a commit that does other things. If you want to propose removing blank lines, do it in a separate PR.